opsdroid/loader.py

```diff
@@ -321,14 +336,18 @@ def _load_modules(self, modules_type, modules):
                 config["module"] = module.get("module", '')
             config["type"] = modules_type
             config["is_builtin"] = self.is_builtin_module(config)
+            if config["name"] in entry_points:
+                config["entrypoint"] = entry_points[config["name"]]
             config["module_path"] = self.build_module_import_path(config)
             config["install_path"] = self.build_module_install_path(config)
             if "branch" not in config:
                 config["branch"] = DEFAULT_MODULE_BRANCH
 
             # If the module isn't builtin, or isn't already on the
             # python path, install it
-            if not (config["is_builtin"] or config["module"]):
+            if not (config["is_builtin"]
```

Comment:
This statement seems to have a mix of `.get` and `[]` notations for the config dictionary. Could you please make it consistent (probably with `.get`).

Reply:
The mix is intentional to accommodate missing config `entrypoint` key, since it only exists when evaluating a packaged extension hooked in via entry point. But it just now occurred to me that perhaps we can count on `is_builtin` or `module` key being truthy in all such cases? Meaning that `config[entrypoint]` would never have a chance to fail.

Reply:
Duh, of course we cannot, given the whole point of this check... I guess we could just always set `config["entrypoint"]` then.

Reply:
I would prefer to set it to be explicit and consistent.